pkg/cli/build/build.go

```diff
@@ -199,6 +201,31 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func appendElementalRPMs(ctx *image.Context) {
+	elementalDir := filepath.Join(ctx.ImageConfigDir, "elemental")
```

Comment:
Similar to my comments in your other PR, it feels wrong to me that the directory structure information would be hardcoded in multiple locations. Can we make this a call into `elemental.go` that retrieves the directory instead of hardocoding it here?